Breeder/BR_Util.cpp

```diff
@@ -2838,12 +2864,16 @@ MediaTrack* HwndToTrack (HWND hwnd, int* hwndContext, POINT ptScreen)
 					if (!GetMediaTrackInfo_Value(chktrack, "B_SHOWINTCP"))
 						continue;
 
-					const double ypos = GetMediaTrackInfo_Value(chktrack, "I_TCPY"),
-					             h    = GetMediaTrackInfo_Value(chktrack, "I_TCPH");
+					const int spacerHeight = GetTrackSpacerHeight(chktrack);
```

Comment:
Would be clearer named `spacerSize` / `GetTrackSpacerSize` since it's width in the MCP.

Reply:
roger that